Tanium output fix - Reopened (#6720)
demisto/content: Packs/Tanium/Integrations/Tanium/CHANGELOG.md

## [Unreleased]

> Fixed an issue where the output of some answers was malformed.

Reviewer: ```suggestion
Fixed an issue where the output of some results were malformed.
```
